### PR TITLE
Update install.sh

### PR DESCRIPTION
**Pull request by bindushree16**
https://github.com/Shippable/pm/issues/10667

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -3,11 +3,11 @@
 yum -y update
 sudo yum install -y yum-utils
 sudo yum install -y centos-release-scl
-sudo yum-config-manager --enable rhel-server-rhscl-7.2-rpms
-sudo yum install -y devtoolset-7.2
+sudo yum-config-manager --enable rhel-server-rhscl-7.2*-rpms
```
Comment (ric03uec) [inline install.sh]: does this work on local?

Comment (bindushree16) [reply install.sh]: i checked now its not working

Comment (bindushree16) [reply install.sh]: i check with the star its not working . i rollback to the previous version

Comment (ric03uec) [reply install.sh]: we should not be sending prs like this that we know will fail for sure.

**Pull request merged**